<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmZitatEintragen</class>
 <widget class="QDialog" name="frmZitatEintragen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>367</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>398</width>
    <height>367</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>398</width>
    <height>367</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lblZitat">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Zitat:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRedner">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Redner:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDatum">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>123</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Datum:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblOrgEinheit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>153</y>
     <width>46</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>OrgEinheit:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtZitat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtRedner">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>90</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="deDatum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>15</y>
     <width>401</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitel">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Zitat eintragen&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbMenue">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hauptmen√º</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>321</y>
     <width>411</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pbBeenden">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Beenden</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbSpeichern">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Bilder/Download.png</normaloff>Bilder/Download.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="edtOrgEinheit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
